test/Dialect/TorchConversion/torch-backend-to-linalg-on-tensors-no-contract-check.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: torch-mlir-opt -p 'builtin.module(torch-backend-to-linalg-on-tensors-backend-pipeline{verify=0})' -split-input-file %s | FileCheck %s
```

Comment:
I guess the verification already has some tests?

Reply:
Yes, the neighboring `verify-linalg-on-tensors-backend-contract.mlir` file takes care of this.